test/Microsoft.ML.TestFramework/Datasets.cs

```diff
@@ -600,15 +600,15 @@ public static class TestDatasets
         public static TestDataset mnist28 = new TestDataset()
         {
             name = "mnist28",
-            trainFilename = @"..\MNIST\Train-28x28.txt",
-            testFilename = @"..\MNIST\Test-28x28.txt"
+            trainFilename = @"Train-28x28.txt",
+            testFilename = @"Test-28x28.txt"
         };
 
         public static TestDataset mnistTiny28 = new TestDataset()
         {
             name = "mnistTiny28",
-            trainFilename = @"..\MNIST\Train-Tiny-28x28.txt",
-            testFilename = @"..\MNIST\Test-Tiny-28x28.txt"
+            trainFilename = @"Train-Tiny-28x28.txt",
```

Comment:
>Train [](start = 30, length = 5)

intentional

Reply:
yes. The 'old' path is from the TLC solution structure. 

---
In reply to: [219997779](https://github.com/dotnet/machinelearning/pull/979#discussion_r219997779) [](ancestors = 219997779)